x-pack/plugins/security_solution/public/resolver/store/data/action.ts

```diff
@@ -75,7 +75,15 @@ interface AppReceivedNewExternalProperties {
      * the `_id` of an ES document. This defines the origin of the Resolver graph.
      */
     databaseDocumentID?: string;
+    /**
```

Comment:
This action isn't specific to `data` at all so I'm going to move it to the top level Resolver actions file.